syntax = "proto3";
service Booking{
  rpc GetBookingByUserId (UserId) returns ( stream BookingMessage) {};
  rpc CreateBooking (BookingMessage) returns (BookingMessage) {};
  rpc GetAllBookings (Empty) returns (stream BookingMessage) {};
}


message UserId {
  string id=1;
}

message BookingMessage {
  string userid=1;
  repeated Date date=2;
}


message Date {
  string date=1;
  repeated string movies=2;
}

message Empty {}



